client/src/components/SpeciesTable.js

```diff
@@ -316,6 +335,45 @@ const EditModal = ({
   )
 }
 
+const DeleteDialog = ({
+  speciesEdit,
+  setSpeciesEdit,
+  openDelete,
+  setOpenDelete,
+  deleteSpecies,
+  loadSpeciesList,
+}) => {
+  const handleDelete = async () => {
+    await deleteSpecies({ id: speciesEdit.id })
+    loadSpeciesList()
+    setOpenDelete(false)
+    setSpeciesEdit(undefined)
+  }
+
+  const closeDelete = () => {
+    setOpenDelete(false)
+    setSpeciesEdit(undefined)
+  }
+
+  return (
+    <Dialog
+      open={openDelete}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+    >
+      <DialogTitle id="alert-dialog-title">{`Please confirm you want to delete ?`}</DialogTitle>
```

Comment:
Minor comment on the punctuation (sorry): No need for a `?`

Reply:
Haha that's cool